kinetic/distribution.yaml

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 141: http://ros.org/reps/rep-0141.html
+---
+release_platforms:
```

Comment:
We are not building Fedora packages on the build farm but we need to create the rpm stuff in the bloom process.